app/javascript/app/components/ghg-emissions/ghg-emissions-selectors/ghg-emissions-selectors-data-utils.js

```diff
@@ -9,35 +9,14 @@ const expandRegionToCountries = (iso, regions) => {
   return flatMap(region.members, expandRegionToCountries);
 };
 
-const expandRegionsToCountries = (isos, regions) =>
-  uniq(flatMap(isos, iso => expandRegionToCountries(iso, regions)));
-
-export const getMetricData = (
-  year,
-  column,
-  metricField,
-  calculationData,
-  regions
-) => {
+export const getMetricData = (year, iso, metricField, calculationData) => {
   const getMetricForYearAndRegion = (y, r) =>
     metricField &&
     calculationData &&
     calculationData[y] &&
     calculationData[y][r] &&
     calculationData[y][r][metricField];
-  let metricData = getMetricForYearAndRegion(year, column.iso);
-  // if no metric data for expandable column then use expanded regions to
```

Comment:
Is that not needed anymore? I think that was for regions for which there was no aggregated data.